docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -30,13 +30,28 @@ NOTE: PacketFence PKI is available on the PacketFence repository as the package
 Installing 
 ^^^^^^^^^^
 
-First of all, you will need to install the package, use the following command matching your OS.
+First of all, you have the choice to install the PacketFence PKI on a standalone machine or on the PacketFence machine.
+
+For the standalone you need to open the port 9393 in tcp to be able to reach the PKI. This has to be done in the firewall OS(iptables).
+
+If you install the PKI on a PacketFence server, you need to do the following:
+
+ # vim /usr/local/pf/conf/iptables.conf
+ 
+Uncomment this line and restart the iptables service:
+
+ # -A input-management-if --protocol tcp --match tcp --dport 9393 --jump ACCEPT
+ # bin/pfcmd service iptables restart
```

Comment:
Specify full path to pfcmd.
You cannot assume the user is in /usr/local/pf as you have not mentioned it anywhere.